src/backend/marsha/core/utils/medialive_utils.py

```diff
@@ -322,3 +322,94 @@ def delete_aws_element_stack(video):
     medialive_client.delete_input(
         InputId=video.live_info.get("medialive").get("input").get("id")
     )
+
+
+def _get_items(  # nosec
+    get_items, items_key, params=None, next_token=None, next_token_key="NextToken"
```

Comment:
```suggestion
    get_items, items_key, params={}, next_token=None, next_token_key="NextToken"
```
With this you can maybe remove the `if` statement line 354 ?

Reply:
Unfortunately, not :

> /app/src/backend/marsha/core/utils/medialive_utils.py:328:34: B006 Do not use mutable data structures for argument defaults.  They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.